cfme/services/catalogs/orchestration_template.py

```diff
@@ -1,166 +1,284 @@
+                   'content': content})
+        view.add_button.click()
+        view.flash.assert_no_error()
+        view.flash.assert_message('Orchestration Template '
+                                  '"{}" was saved'.format(self.template_name))
 
     def update(self, updates):
-        sel.force_navigate('edit_template',
-                           context={'template_type': self.template_type,
-                                    'template_name': self.template_name})
-        fill(create_template_form, {'template_name': updates.get('template_name', None),
-                                    'description': updates.get('description', None)},
-             action=create_template_form.edit_button)
-        flash.assert_success_message('Orchestration Template "{}" was saved'.format(
+        view = navigate_to(self, "Edit")
+        view.fill({'description': updates.get('description', None),
+                   'name': updates.get('template_name', None)})
+        view.save_button.click()
+        view.flash.assert_message('Orchestration Template "{}" was saved'.format(
             self.template_name))
 
     def delete(self):
-        sel.force_navigate('select_template',
-                           context={'template_type': self.template_type,
-                                    'template_name': self.template_name})
-        cfg_btn("Remove this Orchestration Template", invokes_alert=True)
-        sel.handle_alert()
-        flash.assert_success_message('Orchestration Template "{}" was deleted.'.format(
+        view = navigate_to(self, "TemplateDetails")
+        view.configuration.item_select("Remove this Orchestration Template", handle_alert=True)
+        view.flash.assert_message('Orchestration Template "{}" was deleted.'.format(
             self.template_name))
 
     def delete_all_templates(self):
-        sel.force_navigate('orch_template_type',
-                           context={'template_type': self.template_type})
+        view = navigate_to(self, "TemplateType")
         sel.click(pg.check_all())
-        cfg_btn("Remove selected Orchestration Templates", invokes_alert=True)
-        sel.handle_alert()
+        view.configuration.item_select("Remove selected Orchestration Templates", handle_alert=True)
 
     def copy_template(self, template_name, content):
-        sel.force_navigate('select_template',
-                           context={'template_type': self.template_type,
-                                    'template_name': self.template_name})
-        cfg_btn("Copy this Orchestration Template")
-        fill(create_template_form, {'template_name': template_name,
-                                    'content': content},
-             action=create_template_form.add_button)
-        flash.assert_success_message('Orchestration Template "{}" was saved'.format(
+        view = navigate_to(self, "CopyTemplate")
+        view.fill({'name': template_name,
+                   'content': content
```

Comment:
Is there a reason they shouldn't be here? Or do you mean yes, you'll add them to the dict? :)